go/proxyd/server.go

```diff
@@ -167,9 +168,10 @@ func (s *Server) HandleRPC(w http.ResponseWriter, r *http.Request) {
 				continue
 			}
 
-			batchRes[i] = s.handleSingleRPC(ctx, req)
```

Comment:
If I understand this correctly, whether or not the final request in the batch is cached will determine the caching status?

Reply:
meant that to have a single request in the batch to determine the caching status not the last. will fix.